packages/config/.configref

```diff
@@ -1 +1 @@
-25375
\ No newline at end of file
+25374
```

Comment:
This probably shouldn't be decreasing? @TurtIeSocks

Reply:
It is, because one less character in default.json :)

Reply:
Though ideally PRs shouldn't be changing this to help avoid merge conflicts. It will update automatically whenever something is merged/committed to dev or main.